### PR TITLE
[example-project] Adding protein viewer example

### PR DESCRIPTION
**Pull request by kristofferj**

### CODE REVIEW

File: packages/example-studio/components/ProteinInput/ProteinInput.js [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+    if (!newCamera) {
+      this.handleResetCamera()
+    }
+  }
+
+  fetchPdb = id => {
+    const url = `http://www.rcsb.org/pdb/files/${id}.pdb`
+
+    this.setState({
+      loading: true
```
Comment (bjoerge) [inline packages/example-studio/components/ProteinInput/ProteinInput.js]: Rename to `isLoading`?

File: packages/example-studio/components/ProteinInput/ProteinInput.js [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+      pdb: (props.value && props.value.pdb) || '1R6A',
+      newPdb: false
+    }
+  }
+
+  componentDidMount() {
+    this.structure = ''
+    this.fetchPdb(this.state.pdb)
+    // this.loadPDB('1r6a')
+    this.viewer = new Viewer(this._viewerElement, options)
```
Comment (bjoerge) [inline packages/example-studio/components/ProteinInput/ProteinInput.js]: Is there any way to dispose the viewer on `componentWillUnmount`?

File: packages/example-studio/components/ProteinInput/ProteinInput.js [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+    this._viewerElement.addEventListener('mousemove', this.mouseMoveHandler)
+    this._viewerElement.addEventListener('mousewheel', this.mouseWheelHandler)
```
Comment (bjoerge) [inline packages/example-studio/components/ProteinInput/ProteinInput.js]: Also, these listeners should be removed on `componentWillUnmount`

File: packages/example-studio/components/ProteinInput/ProteinInput.js (hunk 2 above)
Comment (kristofferj) [inline packages/example-studio/components/ProteinInput/ProteinInput.js]: Don't know. I thought it did id automatic.

File: packages/example-studio/components/ProteinInput/ProteinInput.js (hunk 3 above)
Comment (kristofferj) [inline packages/example-studio/components/ProteinInput/ProteinInput.js]: 👍

File: packages/example-studio/components/ProteinInput/ProteinInput.js (hunk 1 above)
Comment (kristofferj) [inline packages/example-studio/components/ProteinInput/ProteinInput.js]: 👍

File: packages/example-studio/components/ProteinInput/ProteinInput.js (hunk 2 above)
Comment (bjoerge) [inline packages/example-studio/components/ProteinInput/ProteinInput.js]: From the source code it looks like its `viewer.destroy()`
https://github.com/biasmv/pv/blob/master/src/viewer.js#L1167

**Pull request merged**